common_test.go

```diff
@@ -28,3 +33,77 @@ func TestComputePayloadAuthenticator(t *testing.T) {
 		}
 	}
 }
+
+func runTestOverVersions(t *testing.T, f func(t *testing.T, version Version)) {
+	for _, version := range KnownVersions() {
+		version := version // capture range variable.
+		t.Run(version.String(), func(t *testing.T) {
```

Comment:
Oh cool, does this mean the test runner will output the right thing as all the sub-tests execute?